syft/workers/base.py

```diff
@@ -994,3 +994,6 @@ def is_framework_supported(cls, framework: str) -> bool:
         :return: True/False
         """
         return framework.lower() in framework_packages
+
+
+BaseWorker.register_message_handlers()
```

Comment:
@youben11 moved the logic to the worker level.
Something like this you meant?

@karlhigley what do you think about this?